schemas/iso19139/src/main/plugin/iso19139/extract-relations.xsl

```diff
@@ -33,7 +33,8 @@
                 xmlns:gco="http://www.isotc211.org/2005/gco"
                 xmlns:gmx="http://www.isotc211.org/2005/gmx"
                 xmlns:xlink="http://www.w3.org/1999/xlink"
-                xmlns:util="java:org.fao.geonet.util.XslUtil"
```

Comment:
@wangf1122 this is required for https://github.com/geonetwork/core-geonetwork/pull/7431/files#diff-8ba6fa3b50cb72ae6c6e65d46bf59e1b4f7a4c2102df0c7f1e1c779ae08db967R134

I tested uploading a thumbnail and it's failing due to this.

Reply:
Thanks for pointing it out. I just realized this namespace is used. I have placed it back.